tests/integration/annotation_import/conftest.py

```diff
@@ -291,6 +291,32 @@ def ontology():
     return {"tools": tools, "classifications": classifications}
 
 
+@pytest.fixture
+def wait_for_label_processing():
+    """
+    Do not use. Only for testing.
+
+    Returns project's labels as a list after waiting for them to finish processing.
+    If `project.labels()` is called before label is fully processed,
+    it may return an empty set
+    """
+
+    def func(project):
```

Comment:
We could also use retry decorators for this type of logic which IMO is cleaner. This is how it might look like.
```
from google.api_core import retry

@pytext.fixture
@retry.Retry(deadline=10)
def wait_for_label_processing(project):
     labels = list(project.labels())
     if len(labels) > 0:
         return labels
     else:
         raise ValueError
```